sbws/globals.py

```diff
@@ -36,6 +36,28 @@
     'SafeLogging': '0',
     'LogTimeGranularity': '1',
     'ProtocolWarnings': '1',
+    # Text and options from
+    # https://stem.torproject.org/tutorials/mirror_mirror_on_the_wall.html#can-i-get-descriptors-from-the-tor-process
+    #
+    # Descriptors have a range of time during which they're valid. To get the
+    # most recent descriptor information, regardless of if Tor needs it or not,
+    # set the following.
+    'FetchDirInfoEarly':  '1',
+    'FetchDirInfoExtraEarly': '1',
+
+    # Tor doesn't need all descriptors to function. In particular...
+    #
+    #   * Tor no longer downloads server descriptors by default, opting
+    #     for microdescriptors instead.
```

Comment:
This comment is wrong, because we set UseMicrodescriptors 0 above.

Reply:
I guess it makes sense to have it in the context of stem, not sbws.
Fixup removed the context.

Reply:
s/context/comment/